schemas/device/pipetteUsage.schema.tpl.json

```diff
@@ -1,121 +1,104 @@
 {
   "_type": "https://openminds.ebrains.eu/ephys/PipetteUsage",
-  "_categories": [
-    "deviceUsage"
-  ],
+  "_extends": "/core/schemas/research/deviceUsage.schema.tpl.json",  
   "required": [
-    "pipette",
     "pipetteSolution"
   ],
   "properties": {
     "anatomicalLocation": {
-      "_instruction": "Add the anatomical entity in which the pipette tip is located.",
-      "_linkedTypes": [
-        "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
-        "https://openminds.ebrains.eu/sands/CustomAnatomicalEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntity",
-        "https://openminds.ebrains.eu/sands/ParcellationEntityVersion"
+      "_instruction": "Add the anatomical entity that describes the semantic location of the pipette tip best.",
```

Comment:
"Add the anatomical entity that semantically best describes the anatomical location of the pipette tip."

Reply:
@lzehl I modified it, please take a look.